nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/state/system-diagnostics/system-diagnostics.effects.ts

```diff
@@ -49,13 +51,24 @@ export class SystemDiagnosticsEffects {
                             }
                         })
                     ),
-                    catchError((error) =>
-                        of(
-                            SystemDiagnosticsActions.systemDiagnosticsApiError({
-                                error: error.error
+                    catchError((errorResponse: HttpErrorResponse) => {
+                        if (request.source === 'cluster-listing') {
+                            return of(
+                                ErrorActions.snackBarError({
```

Comment:
Should this be a `banner` on top of the cluster page since what is failing to load the content for the page (if they are on one of the system diagnostics tabs)?

Reply:
Good point. Probably yes. However I don't think we want to make it conditional based on which tab is open.

Reply:
After further consideration, this presents a potential issue with the cluster listing and the system diagnostics dialog are both open and both showing the same banner error(s). Sticking with snackbar errors in this case until we have contextual banner errors.